src/components/LoginLink.js

```diff
@@ -2,10 +2,18 @@ import React from 'react';
 
 import {config} from '../config/index';
 
-export function LoginLink({children, orgCode, ...props}) {
+export function LoginLink({children, postLoginRedirectURL, orgCode, ...props}) {
+  const params = new URLSearchParams(
+    JSON.parse(
+      JSON.stringify({
+        org_code: orgCode,
+        post_login_redirect_url: postLoginRedirectURL
+      })
+    )
+  );
   return (
     <a
-      href={`${config.apiPath}/login${orgCode ? `?org_code=${orgCode}` : ''}`}
+      href={`${config.apiPath}/login${params ? `?${params.toString()}` : ''}`}
```

Comment:
@peterphanouvong this ternary won't actually do anything as `params` will always be defined even if the object keys are `null` you would end up with `/login?org_code=null&post_login_redirect_url=null` or similar.

Either you want to only add the params to URL search params if they exist, or just be ok that they are empty and ditch the ternary

Reply:
Maybe do this instead
```
let params = new URLSearchParams();
orgCode &&  params.append("org_code", orgCode);
postLoginRedirectURL && params.append("post_login_redirect_url", postLoginRedirectURL);
```

then the ternary should behave as expected